tests/ses/deepsea_testsuite.pm

```diff
@@ -34,18 +34,25 @@ sub run {
         systemctl 'status salt-minion';
         barrier_wait {name => 'salt_minions_connected', check_dead_job => 1};
         assert_script_run 'salt-key --accept-all --yes';
+        # give salt time to be ready
+        sleep 10;
```

Comment:
have you seen tests/console/salt.pm how it is done there? Maybe you want to apply the same